docs/manual/advanced-features.tex

```diff
@@ -1006,6 +1006,17 @@
 the method has no side effects, so calling it does not invalidate any
 dataflow facts.
 
+The \refqualclass{dataflow/qual}{SideEffectsOnly} annotation
```

Comment:
This very long paragraph comes in the introduction to a section that very briefly introduces the notions of side effects and determinism.  Documenting these facts is valuable, but this is not the place to do it because it interrupts the flow.  Here, at most a single brief sentence added at the end of the previous paragraph is appropriate.